examples/text_labels_and_annotations/arrow_demo.py

```diff
@@ -235,73 +189,18 @@ def draw_arrow(pair, alpha=alpha, ec=ec, labelcolor=labelcolor):
     'CA': 0.2,
     'GA': 0.1,
     'GT': 0.4,
-    'GC': 0.1}
-
-extreme_data = {
-    'A': 0.75,
-    'T': 0.10,
-    'G': 0.10,
-    'C': 0.05,
-    'AT': 0.6,
-    'AC': 0.3,
-    'AG': 0.1,
-    'TA': 0.02,
-    'TC': 0.3,
-    'TG': 0.01,
-    'CT': 0.2,
-    'CG': 0.5,
-    'CA': 0.2,
-    'GA': 0.1,
-    'GT': 0.4,
-    'GC': 0.2}
-
-sample_data = {
-    'A': 0.2137,
-    'T': 0.3541,
-    'G': 0.1946,
-    'C': 0.2376,
-    'AT': 0.0228,
-    'AC': 0.0684,
-    'AG': 0.2056,
-    'TA': 0.0315,
-    'TC': 0.0629,
-    'TG': 0.0315,
-    'CT': 0.1355,
-    'CG': 0.0401,
-    'CA': 0.0703,
-    'GA': 0.1824,
-    'GT': 0.0387,
-    'GC': 0.1106}
+    'GC': 0.1,
+}
 
 
 if __name__ == '__main__':
-    from sys import argv
-    d = None
-    if len(argv) > 1:
-        if argv[1] == 'full':
-            d = all_on_max
-            scaled = False
-        elif argv[1] == 'extreme':
-            d = extreme_data
-            scaled = False
-        elif argv[1] == 'realistic':
-            d = realistic_data
-            scaled = False
-        elif argv[1] == 'sample':
-            d = sample_data
-            scaled = True
-    if d is None:
-        d = all_on_max
-        scaled = False
-    if len(argv) > 2:
-        display = argv[2]
-    else:
-        display = 'length'
-
     size = 4
-    plt.figure(figsize=(size, size))
+    fig = plt.figure(figsize=(3 * size, size), constrained_layout=True)
+    axs = fig.subplot_mosaic([["length", "width", "alpha"]])
```

Comment:
I find it really hard to learn these concepts from the demo (and I feel like @jklymak may also have opinions here). 

Is there a way to make it clearer, especially given how much code is in this demo? Either by making the different length, width, alpha values very dramatic, and/or adding the values to the axes label?

Reply:
That size is also used later to scale the text size, so I left it in as a variable.